taskweaver/memory/type_vars.py

```diff
@@ -1,4 +1,4 @@
 from typing import Literal
 
-RoleName = Literal["User", "Planner", "CodeInterpreter"]
+RoleName = Literal["User", "Planner", "CodeInterpreter", "Unknown"]
```

Comment:
oh, that's added as a placeholder for `None` to allow proper typing when a Post is newly created yet have not set the send_to at that time. Since Unknown is in the same type as string, many code where expect a str won't need to check `is not None`. When a post is created yet there has some exception, then this value might be save to memory.